src/__tests__/_initTestEnv.js

```diff
@@ -6,7 +6,12 @@
  *
  */
 
+import './raf-polyfill';
+
+/* eslint-disable import/first */
```

Comment:
?

Reply:
Det kreves at man importerer polyfillen først for å få det til å fungere, derfor deaktiveres regelen om at relative importer må komme før absolutte først, ellers klager linteren om det.